cloudformation/membership-attribute-service.yaml

```diff
@@ -328,6 +328,11 @@ Resources:
         FromPort: '443'
         ToPort: '443'
         CidrIp: 0.0.0.0/0
+  MembersDataApiLogGroup:
+    Type: "AWS::Logs::LogGroup"
+    Properties:
+      LogGroupName: !Sub members-data-api-${Stage}
+      RetentionInDays: 30
```

Comment:
Please could raise another ticket to tweak this to a value no more than 26 days?! David F or John D can explain why!

Reply:
My default recommendation for logs is now 14 days

If you have a particular reason to keep it for longer that is fine, but come and chat with me so I can understand your use case.